drivers/ads1115/ads1115.c

```diff
@@ -0,0 +1,181 @@
+
+/*
+ * Copyright (C) Baptiste Le Duc <baptiste.leduc38@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
```

Comment:
```suggestion
/*
 * SPDX-FileCopyrightText: 2025 Baptiste Le Duc <baptiste.leduc38@gmail.com>
 * SPDX-License-Identifier: LGPL-2.1-only
 */
```

We are currently in the process of migrating the copyright statements to the SPDX format (see #21515), so perhaps you can adapt the files as in the suggestion.

The advantage of the SPDX format is, that it can be parsed automatically by software.